R/table_methods.R

```diff
@@ -14,18 +14,55 @@ read_cnt.ConnectorDatabricksTable <- function(connector_object, name, ...) {
 }
 
 #' @description
-#' * [ConnectorDatabricksTable]: Reuses the [connector::read_cnt()] method for [connector::connector_dbi],
-#' but always sets the `catalog` and `schema` as defined in when initializing the connector.
+#' * [ConnectorDatabricksTable]: Creates temporary volume to write object as a parquet file and then
+#' convert it to a table.
 #'
 #' @rdname write_cnt
+#' @param method [ConnectorDatabricksTable]: Which method to use for writing the table. Options:
+#' \itemize{
+#'   \item `volume` - using temporary volume to write data and then convert it to a table.
+#' }
+#'
 #' @export
-write_cnt.ConnectorDatabricksTable <- function(connector_object, x, name, ...) {
-  name <- DBI::Id(
-    catalog = connector_object$catalog,
-    schema = connector_object$schema,
-    table = name
-  )
-  NextMethod()
+write_cnt.ConnectorDatabricksTable <- function(
+    connector_object,
+    x,
+    name,
+    overwrite = FALSE,
+    ...,
+    method = "volume") {
+  checkmate::assert_character(name)
+  checkmate::assert_choice(method, c("volume"), null.ok = FALSE)
+  if (method == "volume") {
```

Comment:
an idea : having a function call write_databricks_volume_method or something like this to wrap this code and we can export it as a function :) what do you think ?